src/chainparams.cpp

```diff
@@ -210,10 +210,18 @@ class CMainParams : public CChainParams {
                         Update(EUpdate::DEPLOYMENT_V17, std::string("v17"), 0, 4032, 419328, 1, 3, 1, false,
                                VoteThreshold(80, 60, 5), VoteThreshold(0, 0, 1), false, 427392)
                 );
-        // updateManager.Add
-        // (
-        //     Update(EUpdate::ROUND_VOTING, std::string("Round Voting"), 7, 100, 100000, 5, 10, 5, false, VoteThreshold(85, 85, 1), VoteThreshold(95, 95, 1))
-        // );
+        updateManager.Add(
```

Comment:
LGTM:  Start is ~16 days, 1 week average must be >= 85%, 1 week grace period after lock in.